src/sentry/eventstore/processing/base.py

```diff
@@ -38,6 +38,9 @@ def exists(self, event: Event) -> bool:
         return self.get(key) is not None
 
     def store(self, event: Event, unprocessed: bool = False) -> str:
+    if not event.get('event_id') or not event.get('project'):
+        logger.error(f\"store method called with event missing 'event_id' or 'project'. Event data: {event}\")
```

Comment:
Not sure if we have a policy around logging every error event in the errors pipeline, but this seems risky